spec/acceptance/compose_v3_spec.rb

```diff
@@ -44,28 +45,48 @@ class { 'docker::compose':
     end
 
     it 'should find a docker container' do
-      shell('docker inspect tmp_compose_test_1', :acceptable_exit_codes => [0])
+      shell('docker inspect web_compose_test_1', :acceptable_exit_codes => [0])
     end
   end
 
-  context 'Destroying compose v3 projects' do
+  context 'creating compose projects with multi compose files' do
```

Comment:
this will fail on windows since there is just one compose file being created in spec_helper_acceptance